examples/README.md

```diff
@@ -73,9 +112,15 @@ Example of an open multi-lane network with human-driven vehicles.
 
 ![](https://raw.githubusercontent.com/flow-project/flow/master/docs/img/highway.gif)
 
+### highway_ramps.py
+
+Example of a highway section network with on/off ramps
+
+![](picture to be added)
```

Comment:
Can you make sure there are issues / other PRs for these TODOs

Reply:
Yeah sure. I am making new PR for most of the TODOs, like more important stuff such as tests and benchmarks.